README.md

```diff
@@ -1,2 +1,104 @@
 # micromasters
 Portal for learners and course teams to access MITx micromasters programs
+
+Getting Started
+---------------
+
+You can either run micromasters locally with a default sqlite database after
+installing the ``requirements.txt`` file, or if you have Docker and
+prefer a cleaner environment, install ``docker-compose`` with ``pip
+install docker-compose`` and run ``docker-compose up``. This will set
+up a near production-ready containerized development environment that
+runs migrations, with the Django development server running on
+port ``8079``.
+
+To run one-off commands, like shell, you can run::
+
+  docker-compose run web python manage.py shell
+
+or to create root user::
+
+  docker-compose run web python manage.py createsuperuser
+
+
+For OS X Development
+--------------------
+
+Due to issues using Docker in OSX, development is more efficient if you
+install `docker-osx-dev <https://github.com/brikis98/docker-osx-dev>`_.
+``docker-osx-dev`` keeps host file system changes synced to the Docker
+container. You can install it by typing ``make`` once you have created the
+Docker container.
+
+Subsequently, before you start your Docker container with
+``docker-compose up``, you would run::
+  
+     docker-osx-dev -m default -s ./
+
+(Assuming your Docker VM is called ``default``, and your current working
+directory is the root of the ``teachersportal`` source directory).
+
+This starts a process that monitors the file system for changes. On startup
+you may receive this error::
+  
+      [ERROR] Found VirtualBox shared folders on your Boot2Docker VM. These may
+      void any performance benefits from using docker-osx-dev:
+      
+      /Users
+      
+      [INSTRUCTIONS] Would you like this script to remove them?
+      1) yes
+      2) no
+
+Answer ``yes``.
+
+Configuration and Start-up
+--------------------------
+
+1. Copy the ``.env.sample`` file to ``.env`` and edit these values::
+    
+    STATUS_TOKEN=
+    EDXORG_BASE_URL=
+    EDXORG_CLIENT_ID=
+    EDXORG_CLIENT_SECRET=
+
+#. (OSX only) Create your docker machine::
+
+     docker-machine create default
+     docker-machine start default
+     docker-machine env default
+     eval "$(docker-machine env default)"
+
+   These commands create a Docker container named ``default``, start the
+   container, and configure environment variables to facilitate communication
+   with ``ccxcon``.
+
+#. Create a user who has permission to view the course::
+
+     docker-compose run web python manage.py createsuperuser
+
+#. Start the machine::
+
+     docker-compose up
+
+#. visit: http://192.168.99.100:8075/
+
+.. note:: (OSX only)
+
+  Your IP address may vary depending on what address docker assigns to your VM.
```

Comment:
Technically it's the VM that's requesting the IP address, not docker. Can you change this to `depending on which IP address your VM uses`?